server/sandbox.go

```diff
@@ -172,6 +172,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		if err != nil {
 			return nil, err
 		}
+		g.SetProcessSelinuxLabel(processLabel)
 	}
```

Comment:
OCID carries a list of MCS Labels that are currently in use in memory.  When it creates a new container with InitLabel() It looks for an unused MCS label and reserves it.  
If you stop OCID and restart it, it needs to figure out ALL of the containers that are created (Whether they are running or not) and repopulate the table of Used MCS Labels.  

InitLabels automatically updates the Table.

We need ReserveLabel any time OCID starts for each container image, and we need UnreserveLabel any time a container is Deleted.